components/engine/src/errors.rs

```diff
@@ -19,11 +22,22 @@ quick_error! {
                 hex::encode_upper(&key), region_id, hex::encode_upper(&start), hex::encode_upper(&end)
             )
         }
-        Protobuf(err: protobuf::ProtobufError) {
+        Protobuf(err: ProtobufError) {
             from()
             cause(err)
             description(err.description())
-            display("Protobuf {}", err)
+        }
+        #[cfg(feature = "prost-codec")]
+        ProstDecode(err: prost::DecodeError) {
+            cause(err)
```

Comment:
maybe adding a `from()` is enough so that you don't need to manually implement the `From`.

Reply:
It doesn't work - because of the way quick_error generates code, the the impls do not have `cfg` and so there is an error when we build with Rust